test/config_yaml_files/agent_tests_values.yaml

```diff
@@ -0,0 +1,31 @@
+clusterName: sck-otel
+splunkPlatform:
+  endpoint:
+  insecureSkipVerify: true
+#  token: ${CI_SPLUNK_HEC_TOKEN}
```

Comment:
Could you leave a comment for this?

Reply:
Never mind, it is [documented](https://github.com/signalfx/splunk-otel-collector-chart/blob/6d3ca53d1e70f60a9c166217dd509a8a376cc058/test/README.md?plain=1#L39)